packages/core/package.json

```diff
@@ -47,5 +47,8 @@
     "http-proxy": "1.18.1",
     "local-ip-url": "1.0.3",
     "path-to-regexp": "6.2.0"
+  },
+  "devDependencies": {
+    "minimist": "^1.2.5"
```

Comment:
@vanpipy 

```diff
- "devDependencies": {
-    "minimist": "^1.2.5"
- }
+ "dependencies": {
+    "minimist": "^1.2.5"
      ....
+ }
```

Reply:
Remove the `devDependencies` and move the `miniimst` into `dependencies` already.